picard/ui/ui_options_general.py

```diff
@@ -98,4 +118,6 @@ def retranslateUi(self, GeneralOptionsPage):
         self.groupBox_2.setTitle(_("General"))
         self.analyze_new_files.setText(_("Automatically scan all new files"))
         self.ignore_file_mbids.setText(_("Ignore MBIDs when loading new files"))
+        self.import_settings.setTitle(_("Importing"))
+        self.label_import.setText(_("Maximum number of files to import without error:"))
```

Comment:
It isn't an error, but a warning, so i would rephrase to: "Warn if user tries to load more than this number of files at once:" (but i'm not a native english speaker).